pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -179,8 +182,23 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
-	if _, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), hostNetworkTargetService, metav1.CreateOptions{}); err != nil {
-		return err
+	// disable the poller until can get the image sorted out.
+	// TODO re-enable
+	if false {
+		podNetworkServicePollerDep.Spec.Replicas = &numNodes
+		podNetworkServicePollerDep.Spec.Template.Spec.Containers[0].Env = []corev1.EnvVar{
+			corev1.EnvVar{
+				Name:  "SERVICE_CLUSTER_IP",
+				Value: service.Spec.ClusterIP,
+			},
+			corev1.EnvVar{
+				Name:  "PORT",
+				Value: fmt.Sprintf("%d", service.Spec.Ports[0].Port),
```

Comment:
This port is always 80, right?

Reply:
as long as the service that we are polling against is set that way